src/components/TimedToast/TimedToast.tsx

```diff
@@ -0,0 +1,60 @@
+// components/TimedToast.tsx
+
+import { useState, useEffect } from 'react';
+
+interface TimedToastProps {
+  //apiUrl: string;
+  id: string,
+  message: string;
+  timeout: number; // Timeout in miliseconds
+}
+
+const TimedToast: React.FC<TimedToastProps> = ({ id, message, timeout }) => {
```

Comment:
its great this implementation, The only recommendation I would make is regarding the use of interfaces. They are used to model objects in this case, but it would be interesting to validate if it is strictly necessary or if it is good practice in React to model it this way. I notice that interfaces are only used here, and it would be helpful to determine if it is necessary to use them more frequently and what the correct approach is for handling interfaces